core/src/main/java/org/frankframework/stream/Message.java

```diff
@@ -381,7 +381,7 @@ public boolean requiresStream() {
 	}
 
 	@Override
-	public void close() throws IOException {
+	public void close() {
```

Comment:
Dit is de echte change, de rest is weghalen van catch-clauses die niet meer geldig zijn hierdoor.

Zoals je ziet, ik heb alleen maar een catch-clause weggehaald en geen echte wijzigingen in de implementatie, dus deze catch-clauses zouden toch al nooit getriggered kunnen worden.